pages/agent/v3/cli_oidc.md

```diff
@@ -7,6 +7,16 @@ See the [OpenID Connect Core documentation](https://openid.net/specs/openid-conn
 
 <%= render "agent/v3/help/oidc_request_token" %>
 
+## OIDC URLs
+
+If using a plugin, such as our [AWS assume-role-with-web-identity](https://github.com/buildkite-plugins/aws-assume-role-with-web-identity-buildkite-plugin) plugin, you'll need to provide an OpenID provider URL, the provider URL should be set to https://agent.buildkite.com.
```

Comment:
General tightening. We can't always, but it's best to avoid "our" because it's unclear who it includes.

```suggestion
If using a plugin, such as the [AWS assume-role-with-web-identity](https://github.com/buildkite-plugins/aws-assume-role-with-web-identity-buildkite-plugin) plugin, you'll need to provide an OpenID provider URL. You should set the provider URL to:  https://agent.buildkite.com.
```